### PR TITLE
chore: update variables5.rs book link

### PR DESCRIPTION
**Pull request by saurav-2104**
chore: update variables5.rs book link

### CODE REVIEW

**Comment by saurav-2104** [comment]
As a first time user, I was not able to figure out which book the exercise was talking about.

File: info.toml [hunk 1]
```diff
@@ -52,7 +52,8 @@ because we want to assign a different typed value to an existing variable. Somet
 you may also like to reuse existing variable names because you are just converting
 values to different types like in this exercise.
 Fortunately Rust has a powerful solution to this problem: 'Shadowing'!
-You can read more about 'Shadowing' in the book's section 'Variables and Mutability'.
+You can read more about 'Shadowing' in the book's section 'Variables and Mutability': 
+https://doc.rust-lang.org/book/ch03-01-variables-and-mutability.html?highlight=variables#shadowing.
```
Comment (jrvidal) [inline info.toml:56]: Hm, do we want the `highlight` query param?

Comment (saurav-2104) [reply info.toml]: Oh, my bad. I think I copied my searched results from browser. Removed it.

File: info.toml [hunk 2]
```diff
@@ -52,7 +52,8 @@ because we want to assign a different typed value to an existing variable. Somet
 you may also like to reuse existing variable names because you are just converting
 values to different types like in this exercise.
 Fortunately Rust has a powerful solution to this problem: 'Shadowing'!
-You can read more about 'Shadowing' in the book's section 'Variables and Mutability'.
+You can read more about 'Shadowing' in the book's section 'Variables and Mutability': 
+https://doc.rust-lang.org/book/ch03-01-variables-and-mutability.html#shadowing.
```
Comment (jrvidal) [inline info.toml:56]: One last minor nit: in my terminal, the final dot messes with the link, so it ends up not jumping to the "Shadowing" section.

Comment (saurav-2104) [reply info.toml]: Removed "."
Also, `cargo` updated the Cargo.lock as well. At this point, I am not sure if it is unwanted.

**Pull request merged**